packages/e2e/cypress/integration/savedDashboards.spec.ts

```diff
@@ -10,25 +10,11 @@ describe('Dashboard List', () => {
         Cypress.Cookies.preserveOnce('connect.sid');
     });
 
-    it('Should display dashboards', () => {
-        cy.visit('/');
-        cy.findByRole('button', { name: 'Browse' }).click();
-        cy.findByRole('button', { name: 'Dashboards' }).click();
-        cy.findByText('Jaffle dashboard').should('exist');
-    });
-
-    it('Should delete dashboards', () => {
-        cy.visit('/');
-        cy.findByRole('button', { name: 'Browse' }).click();
-        cy.findByRole('button', { name: 'Dashboards' }).click();
-        // click on delete
-        cy.get('[data-icon=more]').click();
-        cy.findByRole('button', { name: 'Delete' }).click();
-        // click on delete in the popup
-        cy.get('[data-cy=submit-base-modal]').click();
-        cy.findByText('No results available');
-    });
-
+    /* 
+        These tests assumes there are no dashboards, 
+        If we want to make it compatible with multiple dashboards,
+        we should filter the buttons for updating and deleting by selecting only 1
+    */
```

Comment:
When the env is created we seed a dashboard, thats why we were checking if it existed, deleting it, and creating a new one with the same name. This way we could run the e2e tests multiple times without failing.

Reply:
If we want to change this order, we can stop relying on the seeded dashboard and change the tests to handle multiple dashboards.